src/main.ts

```diff
@@ -145,6 +147,7 @@ async function executeGame(cxt: BrowserContext, shouldRecord:boolean = false) {
     return { time: time, expression: cxt.expression, charCount: cxt.cnt, playURL: cxt.rawLevelUrl, level: cxt.level, gameplay: gameplay } as ScoringResult
   }  
   catch (e) {
+    metrics.increment("game.execute.error", 1);
```

Comment:
we want this to be timeout error like `game.execute.timeout`

Reply:
I'd be fine w/ having both.  1 general one for all errors (this line), and one emitted specifically for TimeoutErrors in addition to this one.